src/TestInfrastructure/Framework/ConfigurableVsProjectSystemHelper.cs

```diff
@@ -52,7 +52,8 @@ public Project GetSolutionFolderProject(string solutionFolderName, bool createOn
 
         public ProjectItem FindFileInProject(Project project, string fileName)
         {
-            throw new NotImplementedException();
+            var item = project.ProjectItems.OfType<ProjectItem>().FirstOrDefault(pi => StringComparer.OrdinalIgnoreCase.Equals(pi.Name, fileName));
+            return item;
```

Comment:
Fix the mock implementation to make the method work sufficiently for the new tests.